app/controllers/admin/api/accounts_controller.rb

```diff
@@ -258,9 +262,9 @@ def find_buyer_account
       buyer_users.find_by!(username: username).account
     when user_id = params[:user_id]
       buyer_users.find(user_id).account
-    when current_account.master? && provider_key = params[:buyer_provider_key]
```

Comment:
There is no such developer provider key ... So not really needed to remove this check.
The idea behind is if I understand for documentation only?

Either we duplicate this logic/controller into https://github.com/3scale/porta/tree/121a66ca7566cbb33e5ed9460efb9ae007f43cba/app/controllers/master/api

Duplicating would be a bad idea as many APIs are shared between provider and master
Another solution is to have a different json file for documentation served depending if you are master or provider but that is overkill IMHO. https://github.com/3scale/porta/blob/121a66ca7566cbb33e5ed9460efb9ae007f43cba/app/controllers/api_docs/services_controller.rb#L59-L65